helpers/github.ts

```diff
@@ -112,6 +113,7 @@ export async function getRepoUrls(orgOrRepo: string) {
         } else console.warn(`Getting repo ${params[0]}/${params[1]} failed: ${res.status}`);
       } catch (e: unknown) {
         console.warn(`Getting repo ${params[0]}/${params[1]} failed: ${e}`);
+        throw e;
```

Comment:
```suggestion
        throw error;
```

Reply:
Done in a new commit. Rename was needed in at least 2 places.